e2e/data-factory/persistence/adapters/api.ts

```
@@ -0,0 +1,99 @@
+import type {PersistenceAdapter} from '../adapter';
+
+interface ApiAdapterOptions {
+    agent: any; // HTTP agent (e.g., supertest agent)
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)